src/scripts/components/foundations/Text.js

```diff
@@ -10,6 +10,8 @@ export const ColorText = css`
       _color = props.theme.colors.button.purple
     } else if (props.pink) {
       _color = props.theme.colors.button.pink
+    } else if (props.green) {
+      _color = props.theme.colors.button.green
```

Comment:
Same here, why not just pass in the color value directly?

Reply:
Could we change that in another PR?